device_services/generic/barrier_control.md

```diff
@@ -11,24 +11,29 @@ The service represent devices like garage doors, barriers, window protection sha
+| Type | Interface                | Value type | Properties | Description                                                                                                                                                                              |
+|------|--------------------------|------------|------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| in   | cmd.notiftype.get_report | null       |            | Requests notification types the device supports when signalling opening/closing of the door.                                                                                             |
+| in   | cmd.notiftype.set        | bool_map   |            | Sets notification type the device is using when signalling opening/closing of the door. Configurable notification types are defined in [`sup_notiftypes`](#service-properties) property. |
+| out  | evt.notiftype.report     | bool_map   |            | Returns notification types the device is using while opening/closing door.                                                                                                               |
+| -    |                          |            |            |                                                                                                                                                                                          |
+| in   | cmd.op.stop              | null       |            | Stops any operation immediately.                                                                                                                                                         |
+| -    |                          |            |            |                                                                                                                                                                                          |
+| in   | cmd.state.get_report     | null       |            | Requests the current state of the device.                                                                                                                                                |
+| out  | evt.state.report         | string     | `position` | Reports the current state of the device, one of values defined in [`sup_states`](#service-properties) property.                                                                          |
+| -    |                          |            |            |                                                                                                                                                                                          |
+| in   | cmd.tstate.set           | string     | `stop_at`  | Sets the target state of the device to the one of values defined in [`sup_tstates`](#service-properties) property.                                                                       |
```

Comment:
`stop_at` -> lets call 'positions' as well as it releates to the same thing as property in evt.state.report

Reply:
Ok